minifi-commons/minifi-commons-schema/src/main/java/org/apache/nifi/minifi/commons/schema/serialization/SchemaLoader.java

```diff
@@ -78,6 +80,27 @@ public static ConfigSchema loadConfigSchemaFromYaml(Map<String, Object> yamlAsMa
 
     public static ConvertableSchema<ConfigSchema> loadConvertableSchemaFromYaml(Map<String, Object> yamlAsMap) throws SchemaLoaderException {
         String version = String.valueOf(yamlAsMap.get(ConfigSchema.VERSION));
+        if (StringUtil.isNullOrEmpty(version) || String.valueOf((Object) null).equals(version)) {
```

Comment:
@JPercivall sorry, I thought you were wanting behavior like that in regards to [your comment above.](https://github.com/apache/nifi-minifi/pull/45#discussion_r85137468)

I must've misread the comment, now it's looking more like it may have been automatically addressed by the other validation change which clears the errors before converting.